recipes/xapian-core/all/patches/0001-add-msvc-cl-sh.patch

```diff
@@ -0,0 +1,99 @@
+new file mode 100755
```

Comment:
I think I have seen that already in another recipe(s). please correct me if I am wrong @madebr 
does it make sense to make it standalone package (build requires)?

Reply:
Indeed, I used it first time in re2c.
It does make sense to create a standalone package for it.
But it's not really well tested. I'm also not sure whether some packages need special cases.